content/installation/netcore/install/Supported-technology-netcore.md

```diff
@@ -1,39 +1,94 @@
 
+
 <!--
 title: "Contrast .NET Core Agent Supported Technologies"
 description: "Contrast .NET Core agent supported technologies"
-tags: "installation agent .NET Core windows azure supported technologies"
+tags: "installation agent .NET Core windows linux azure supported technologies"
 -->
 
 ## Platform Support
 
 ### .NET Core for Windows
 
-The Contrast .NET Core agent supports analysis of web applications built on the following technologies when executing within a Windows environment. Refer the the following table for supported versions.
+#### Operating Systems
```

Comment:
@Silvenga I added level 4 subheads in the Windows and Linux sections. Can you make sure that I used the right terms?

Reply:
> Containers

Containers might slightly overloaded here e.g. Containers for Windows (e.g. docker) != Hosting Containers. Maybe "Technologies" would be a better phrase. I don't like the Hosting vs Server containers either, but that's a separate issue 😹 .

We might be able to get away with using Distributions within the Core for Windows section too, since the operating system is implied by "Core for Linux".

e.g. LTSC and SAC are different distribution channels of Windows.